Source/WarpX.H

```diff
@@ -423,6 +438,10 @@ public:
          amrex::IntVect y_nodal_flag, amrex::IntVect z_nodal_flag,
          const int lev);
 
+    // Function to add prtl and cell contributions to costs_heuristic
```

Comment:
Please make this a full-flavored Doxygen string.

Reply:
Documentation added.